bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Shares/SelectionShare.cs

```diff
@@ -156,29 +153,29 @@ public bool Equals(CiphertextDecryptionSelectionShare? other)
         return ObjectId == other.ObjectId &&
                SequenceOrder == other.SequenceOrder &&
                DescriptionHash.Equals(other.DescriptionHash) &&
-               Share.Equals(other.Share); // TODO: proof comparison
+               Share.Equals(other.Share) &&
+               Commitment.Equals(other.Commitment);
     }
 
     public override bool Equals(object? obj)
     {
-        return ReferenceEquals(this, obj) || (obj is CiphertextDecryptionSelectionShare other && Equals(other));
+        return ReferenceEquals(this, obj) || (obj is SelectionShare other && Equals(other));
```

Comment:
not sure if the bitwise and or the text 'and' would be best for this one.  mixing the is and the && might give a strange result.  not sure however.